package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "dependencies": {
     "abortcontroller-polyfill": "^1.7.3",
+    "buffer": "^6.0.3",
```

Comment:
I really don't want to depend on this. Vanila JS shoula already have everything we need. What are you missing?

Reply:
The write* (writeUint32LE, writeUint32BE) functions are quite useful. 

But you're right. I wasn't aware that DataView can handle different endianess too.